test/config/TestConfig.cpp

```diff
@@ -217,6 +216,10 @@ TEST_F(ConfigTestFixture, HappyCaseMinimumConfig)
     ASSERT_STREQ(filePath.c_str(), config.key->c_str());
     ASSERT_STREQ("thing-name value", config.thingName->c_str());
     ASSERT_FALSE(config.fleetProvisioning.enabled);
+    ASSERT_FALSE(config.deviceDefender.enabled);
```

Comment:
Don't we have a `ConfigTestFixture::DefaultFeaturesEnabled` that all of these assertions could be rolled up under? Also could you rename that from `ConfigTestFixture::DefaultFeaturesEnabled` to ``ConfigTestFixture::AssertDefaultFeaturesEnabled`?